feature-libs/organization/administration/components/shared/card/card.component.html

```diff
@@ -29,6 +29,7 @@ <h4>
 
   <div class="main">
     <cx-org-message></cx-org-message>
+    <ng-content select="[messages]"></ng-content>
```

Comment:
```suggestion
    <ng-content select="[info]"></ng-content>
```
let use some other tag